### ISSUE TITLE
is this useful in real-time detection?

### ISSUE THREAD

**Issue by pelre** [zhreshold/mxnet-ssd.cpp#7]
hello, deep learning newbie here.
can this be used in real time detection? and if so, is it API rewriting? which function do I use to try and get it to work?

**Comment by zhreshold** [comment]
see #6

**Comment by pelre** [comment]
thanks, the rewrite api files are in the include folder right?

**Comment by zhreshold** [comment]
Yes, but you may not need to rewrite it, just use it for frame by frame processing.

**Comment by pelre** [comment]
frame by frame processing works for real-time detection as well? woah, I had no idea, so I can get it to detect on a webcam and such?

**Comment by zhreshold** [comment]
Well, if you want real-time, you need a gpu at least.
refer to https://github.com/zhreshold/mxnet-ssd

**Comment by pelre** [comment]
I got a pretty good gpu at the moment, object detection on videos work well, how do I begin with real-time detection? Thanks for being so helpful 👍

**Comment by zhreshold** [comment]
You need to enable GPU in mxnet, but it's not written in makefile right now.

**Comment by pelre** [comment]
oh alright, so all I have to do is enable GPU in the makefile? I'd gladly do so, but the only thing is I can't see the makefile in the directory

**Comment by zhreshold** [comment]
mxnet/amagamation

**Comment by pelre** [comment]
the amalgamation folder is not the windows branch, could it still work if I use the makefile from the master branch?

**Comment by zhreshold** [comment]
the win branch is what I pre-built with visual studio, you can use amalgamation or simply build the entire lib using VS, either one is ok. The latter have an official tutorial on dmlc/mxnet
